### ISSUE TITLE
How can I manually change ALT+F2 and ALT+F1 shortcuts in Mate 1.20.01

### ISSUE THREAD

**Issue by JanuszChmiel** [mate-desktop/mate-control-center#530]
Dear specialists,
Unfortunately for Me, only Ubuntu Bionic support Dbus by The best way, so I can reliably use Mate with Orca screen reader even from Termux from Debian Buster by using Proot.
But mate-keybindings-properties C app do not contain The possibility to change predefined shortcuts ALT+F1 and ALT+F2. I Am afraid, that simply trying to compile newest stable Mate from Ubuntu Bionic is no simply possible. If it would, Ubuntu experts would allready included latest stable Mate to their Ubuntu Bionics ARM 64 Bit repositoryes.
SO I would like to change ALT+F2 and ALT+F1 to.
CTRL+r
and CTRL+m
But where can I change The keys? Did you included some .xml or some other editable files, Or I would had to directly change mate-settings-daemon or other source code manually to change those hot keys?
I have run Mate desktop perfectly on Ubuntu Bionics ARM64 Bit edition including fast and reliably window switching, many included apps are working, even Orca start automatically without crashes, Orca even get focus automatically after executing
mate-session
Thank you very very much for yours advice
Sure, I know, that if I would like to compile whole Mate from Ubuntu sources, I must include Ubuntu src repositoryes and that it will not be easy mission.
What would happen, if I would execute newer mate-keybindings-properites compiled Binary from Debian ARM64 Bit buster? Or I can not mix mate parts by this way, because I will also not be able to simply change hot keys by using newest version of mate-keybindings-properties.
Thank you very muchfor yours support.

**Comment by vkareh** [comment]
If you're talking about using `<Ctrl>M` to open the Main Menu and `<Ctrl>R` to open the Run Dialog, you can already change those by opening the `mate-keybinding-properties` application.

Alternatively you can run these two commands:
```
gsettings set org.mate.Marco.global-keybindings panel-main-menu '<Ctrl>M'
gsettings set org.mate.Marco.global-keybindings panel-run-dialog '<Ctrl>R'
```

**Comment by JanuszChmiel** [comment]
Oh Dear Mr Kareh,
I have tried to do that from app GUI, but really, there are no list items
for changing those keys. There are when I have executed from Debian Buster
but not from Ubuntu Bionic. Thank you very very much for yours fast and
professional answer. Really, yours desktop have brought The joy to my life.
And thank you that you did not coded mate-session so it would not allow
users to use it with XVFB and with X11vnc.
Gnome-session do not support such session types so it is problem. And yours
Mate desktop is really The most accessible and stable one. Thank you for
yours 100 % accessible mate-control center app and many others.
